lib/Core/SpecialFunctionHandler.cpp

```diff
@@ -309,37 +308,37 @@ void SpecialFunctionHandler::handleSilentExit(ExecutionState &state,
                                               KInstruction *target,
                                               std::vector<ref<Expr> > &arguments) {
   assert(arguments.size()==1 && "invalid number of arguments to exit");
-  executor.terminateState(state);
+  executor.terminateStateEarly(state, "", StateTerminationType::SilentExit);
```

Comment:
Can you clang-format these entire tiny functions?

Reply:
Sure.

Reply:
Thanks, but please squash the formatting commit into the first one.

Reply:
Of course, thought there might come up more requests.  Were those all the functions you had in mind?

Reply:
Yes (and the code you changed)

Reply:
For most of the code I kept the surrounding style. Is there any specific section I should reformat?

Reply:
Typically if you touch a tiny function, it's nicer to reformat everything with clang-format.